Tests/Magick.NET.Tests/Shared/Optimizers/ImageOptimizerTests.cs

```diff
@@ -448,6 +448,23 @@ public void Compress_CannotCompressPngFile_FileIsNotSmaller()
             });
         }
 
+        [TestMethod]
+        public void Compress_IgnoreUnsupportedFile_DoesNotThrowException()
```

Comment:
Thanks for following the pattern that is used in this file. Would love to change these to the new unit test structure but that should probably be a separate PR. Would you be willing to change that in a new PR?

Reply:
Is the new unit test structure like [this test class](https://github.com/dlemstra/Magick.NET/blob/7e103d6bfdbcef967d5d9e97ecbb31dd3b5037a9/Tests/Magick.NET.Tests/Windows/Defines/Pdf/PdfReadDefinesTests/ThePasswordProperty.cs#L27), where the class name is "The[ThingBeingTested]" and the method names are "Should[ExpectedBehavior]"? If so yes I can update these tests in a new PR.

Reply:
OK I will follow that pattern in the next PR